lib/Drush/CommandFiles/EvalCommandFile.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Drush\CommandFiles;
+
+/**
+ * @file
+ *   Evaluate PHP code.
+ */
+
+class EvalCommandFile
+{
+  /**
+   * Evaluate arbitrary php code after bootstrapping Drupal (if available).
+   *
+   * @param string $php Code to execute.
+   * @aliases php-eval, eval, ev
+   * @usage php-eval 'variable_set("hello", "world");'
+   *   Sets the hello variable using Drupal API.
+   * @usage php-eval '$node = node_load(1); return $node->title;'
+   *   Loads node with nid 1 and then prints its title.
+   * @usage php-eval "file_unmanaged_copy('$HOME/Pictures/image.jpg', 'public://image.jpg');"
+   *   Copies a file whose path is determined by an environment's variable.
+   *   Note the use of double quotes so the variable $HOME gets replaced by
+   *   its value.
+   * @usage php-eval "node_access_rebuild();"
+   *   Rebuild node access permissions.
+   * @default-format var_export
+   */
+  public function phpEval($php, $options =
+    [
+      'format' => 'var_export',
```

Comment:
I put in both @default-format and `'format' => 'var_export'` because I was experimenting; only one is needed on any given command. If you are using a Symfony Console command (whose parameters are always InputInterface, OutputInterface), then you must use @default-format.

The associative array as a default value, as far as I know, has always worked in php, but I have only verified this as far back as php 5.4.  Its use in this context -- to define the default values for your options -- originated in Robo.